src/Driver/driver_configs.jl

```diff
@@ -252,12 +259,24 @@ function AtmosLESConfiguration(
         meshwarp = meshwarp,
     )
 
+    if cutofforder_horz < polyorder_horz || cutofforder_vert < polyorder_vert
```

Comment:
Do we want a logical OR here or a logical AND? I thought both conditions were needed. Can we have a case where only one of the two is satisfied and we still can apply the filter?

Reply:
The same question applies to the following configurations

Reply:
we only need to apply the filter if the cutoff order is < polyorder

Reply:
Yes, but for both its components, horizontal and vertical, no?

Reply:
if we apply a cutoff filter that is the same degree or higher, it won't do anything (so is safe to apply the filter in the other direction).

Reply:
I don't think so. Since currently there is an `assert` in the `Filter.jl` module. I tested this and got the following stack:

```
ERROR: LoadError: AssertionError: all(0 .<= Nc .<= N)
Stacktrace:
 [1] CutoffFilter(::DiscontinuousSpectralElementGrid{Float64,3,(5, 5, 5),216,Array,Tuple{Array{Float64,1},Array{Float64,1},Array{Float64,1}},Tuple{Array{Float64,2},Array{Float64,2},Array{Float64,2}},Array{Float64,3},Array{Float64,4},Array{Int64,1},Array{Int64,2},Array{Int64,3},ClimateMachine.Mesh.Topologies.StackedCubedSphereTopology{Float64,2},Tuple{Array{Float64,2},Array{Float64,2},Array{Float64,2}}}, ::Tuple{Int64,Int64,Int64}) at /home/valeria/CliMA/ClimateMachine.jl/src/Numerics/Mesh/Filters.jl:264
```

Reply:
@simonbyrne  I fixed this in my branch `valeria/fix-cutoff-degree-cl-option`. You can merge it here when the issue regarding the `apply_async!` is sorted out.

Reply:
Update: I pushed it to this branch. So it is fixed now and I dismissed my request for changes in this PR